docs/src/Atmos/Microphysics.md

```diff
@@ -0,0 +1,369 @@
+v_{drop} = \left(\frac{8}{3 \; C_{drag}} \left( \frac{\rho_{water}}{\rho} -1 \right) \right)^{1/2} g^{1/2} \; r^{1/2} = v_c(\rho) \; g^{1/2} \; r^{1/2}
+\end{equation}
+```
+where:
+ - ``g`` is the gravitational acceleration,
+ - ``C_{drag}`` is the drag coefficient,
+ - ``\rho_{water}`` is the density of water,
+ - ``\rho`` is the density of air.
+
+The mass weighted terminal velocity ``v_t`` is defined following Ogura and Takahashi 1971
+```math
+\begin{equation}
+v_t = \frac{F_{rain}}{RWC}
+\end{equation}
+```
+where:
+ - ``F_{rain} = \int_0^\infty n(r) \; m(r) \; v_{drop}(r) dr`` is the vertical flux of rain drops,
+ - ``RWC = \int_0^\infty n(r) \; m(r) dr = \rho \; q_{rai}`` is the rain water content.
+
+Integrating over the assumed Marshall-Palmer distribution results in
+```math
+\begin{equation}
+RWC = \frac{8 \pi \; n_{0_{MP}} \; \rho_{water}}{\lambda_{MP}^4}
+\label{eq:lambda}
+\end{equation}
+```
+```math
+F_{rain} = \Gamma \left(\frac{9}{2} \right) \frac{4}{3} n_{0_{MP}} \; \pi \; \rho_{water} v_c(\rho) \; g^{1/2} (\lambda_{MP})^{-9/2}
+```
+Eliminating ``\lambda_{MP}`` between the two equations and dividing by ``RWC`` results in
+```math
+\begin{equation}
+v_t = \Gamma \left( \frac{9}{2} \right) \; \frac{v_c(\rho)}{6} \; (8 \pi)^{-1/8} \left( \frac{\rho}{\rho_{water}} \right)^{1/8} n_{0_{MP}}^{-1/8} g^{1/2} q_{rai}^{1/8} = v_{t_{c}}(\rho) \; n_{0_{MP}}^{-1/8} g^{1/2} q_{rai}^{1/8}
+\end{equation}
+```
+The default value of ``C_{drag}`` is chosen such that the ``v_t`` is close to the empirical terminal velocity formulation in Smolarkiewicz and Grabowski 1996. Assuming a constant drag coefficient is an approximation as it should be size and flow dependent, see [drag_coefficient](https://www.grc.nasa.gov/www/K-12/airplane/dragsphere.html).
+
+
+```@example rain_terminal_velocity
+using CLIMA.Microphysics
+using Plots
+
+# eq. 5d in Smolarkiewicz and Grabowski 1996
+# https://doi.org/10.1175/1520-0493(1996)124<0487:TTLSLM>2.0.CO;2
+function terminal_velocity_empirical(q_rai::DT, q_tot::DT, ρ::DT, ρ_air_ground::DT) where {DT<:Real}
+    rr  = q_rai / (DT(1) - q_tot)
+    vel = DT(14.34) * ρ_air_ground^DT(0.5) * ρ^-DT(0.3654) * rr^DT(0.1346)
+    return vel
+end
+
+q_rain_range = range(1e-8, stop=5e-3, length=100)
+ρ_air, q_tot, ρ_air_ground = 1.2, 20 * 1e-3, 1.22
+
+plot(q_rain_range * 1e3,  [terminal_velocity(q_rai, ρ_air) for q_rai in q_rain_range], xlabel="q_rain [g/kg]", ylabel="velocity [m/s]", title="Average terminal velocity of rain", label="CLIMA")
+plot!(q_rain_range * 1e3, [terminal_velocity_empirical(q_rai, q_tot, ρ_air, ρ_air_ground) for q_rai in q_rain_range], label="Empirical")
+savefig("rain_terminal_velocity.svg") # hide
+nothing # hide
+```
+![](rain_terminal_velocity.svg)
+
+## Cloud condensation/evaporation
+
+Condensation and evaporation of cloud water is parameterized as a relaxation to equilibrium. The equilibrium state is calculated using saturation adjustment defined in the `MoistThermodynamics` module.
```

Comment:
If this is time-stepped explicitly, we do not need saturation adjustment, just the equilibrium value at the current time step.

Reply:
correct - changed!